helpers/validators.ts

```diff
@@ -0,0 +1,8 @@
+import { isHexString } from "ethers/lib/utils";
+import { define } from "superstruct";
+
+export function hexString() {
+  return define<`0x${string}`>("hexString", (value) => {
+    return isHexString(value);
+  });
+}
```

Comment:
interesting never seen this used before. so does this enforce 0x at the start?

Reply:
It checks 0x at the start and also check if all nibbles are valid hex and if there are an even number of them I think